source/assets/stylesheets/fonts/filename-generator.sh

```diff
@@ -0,0 +1,23 @@
+#/bin/bash
+# example usage: sh filename-generator.sh <font filename>
+
+set -eu
+
+filename=${1?Usage: $0 <filename>}
+
+md5hash=$( md5 -q "$filename" | cut -c -10 )
+filename_ext=${filename##*.}
+
+echo -n "What is the version number? [default: 1] "
+read -r version
+: "${version:=1}"
+
+echo -n "What type of font (e.g. tabular numbers)? [default: ''] "
```

Comment:
can the examples by only what you should type?

It looks like you'd only want to type 'tabular' here? not 'tabular numbers'?